solc/CommandLineInterface.cpp

```diff
@@ -510,8 +519,13 @@ void CommandLineInterface::createFile(string const& _fileName, string const& _da
 bool CommandLineInterface::parseArguments(int _argc, char** _argv)
 {
 	// Declare the supported options.
-	po::options_description desc(
-		R"(solc, the Solidity commandline compiler.
+	po::options_description desc(R"(solc, the Solidity commandline compiler.
+
+Solidity Copyright (C) 2014-2017 Solidity contributors.
```

Comment:
Perhaps we can do without, but the GPL "how to apply the license" suggests this. The thing is that the copyright is just shared by the authors which I think is the default, so perhaps we can just remove it.